iosxe_iso2vbox.py

```diff
@@ -616,8 +624,12 @@ def main(argv):
     logger.warn('Building Vagrant box')
 
     # Add the embedded Vagrantfile
-    vagrantfile_pathname = os.path.join(
-        pathname, 'include', 'embedded_vagrantfile_xe')
+    if args.virtio==True:
```

Comment:
Spaces around ==.

Maybe a good idea if you run pep8 just to check for common syntax issues?

Reply:
Fixed this one.

Ran pep8, only line-too-long warnings left. Not going to fix them right now.